zeeguu/core/feed_handler/newspaperfeed.py

```diff
@@ -0,0 +1,55 @@
+import newspaper
+from .feed_handler import FeedHandler
+
+from zeeguu.logging import log
+
+
+class NewspaperFeed(FeedHandler):
+    def __init__(self, url: str, feed_type: int, use_cache: bool = True):
+        super().__init__(url, feed_type)
+        self.use_cache = use_cache
+        self.extract_feed_metadata()
```

Comment:
Given that the `extract_feed_metadata` is called in both the subclass inits, it would make sense to move them to the superclass. Something like: 

```
self.url = url
         self.feed_type = feed_type
         self.title = ""
         self.description = ""
         self.image_url_string = ""
         self. extract_feed_metadata()
```

and then this one becomes

```
    def __init__(self, url: str, feed_type: int, use_cache: bool = True):
         self.use_cache = use_cache
         super().__init__(url, feed_type)
```

altough, it's a very very minor observation!

Reply:
Oh that is a good point, since it wasn't implemented in the Super-class I ignored it, but let's do it that way. It's much cleaner!